.github/workflows/ci_python.yml

```diff
@@ -345,20 +420,23 @@ jobs:
         id: deployment
         uses: actions/deploy-pages@v2
 
-  # release:
-  #   name: Release
-  #   runs-on: ubuntu-latest
-  #   if: "startsWith(github.ref, 'refs/tags/')"
-  #   needs: [linux, windows, macos, sdist]
-  #   steps:
-  #     - uses: actions/download-artifact@v3
-  #       with:
-  #         name: wheels
-  #     - name: Publish to PyPI
-  #       uses: PyO3/maturin-action@v1
-  #       env:
-  #         MATURIN_PYPI_TOKEN: ${{ secrets.PYPI_API_TOKEN }}
-  #       with:
-  #         command: upload
-  #         args: --skip-existing *
-  #         working-directory: python
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    environment: pypi
```

Comment:
This should use a GitHub deploy environment. I think that means we need to setup an environment. The main benefit is that since it's considered a "deployment" we can configure manual approval if we want, etc.